DEVDOCS-2002: Finding Apps ID Tutorial (#510)
bigcommerce/dev-docs: docs/api-docs/channels/building-channel-apps.md

    "app": {
      "id": 123,                         // ID of the app
      "sections": [{
        "title": "Overview",             // Label displayed to merchant in nav bar
        "query_path": "overview"         // query param passed app iframe
       }, {
       "title": "Settings",
       "query_path": "settings"
      }]
    }
}]
```

[![Open in Request Runner](https://storage.googleapis.com/bigcommerce-production-dev-center/images/Open-Request-Runner.svg)](https://developer.bigcommerce.com/api-reference/cart-checkout/channels-listings-api/channels/createchannel#requestrunner)

Once created, channels show up in the product list within the control panel so the merchant can choose which products should be available on it. The orders list is updated to include a filter to show which orders came in from the channel. The value of the `name` property is displayed to the merchant in the control panel.

For a complete reference, see [Channels API](https://developer.bigcommerce.com/api-reference/cart-checkout/channels-listings-api/channels/createchannel).

<div class="HubBlock--callout">
<div class="CalloutBlock--info">
<div class="HubBlock-content">

<!-- theme: info -->

### Note

> > * The ID of an app can be found in the URL when editing the app in [DevTools](https://devtools.bigcommerce.com/). For more information, see [Finding an App's ID](https://developer.bigcommerce.com/api-docs/apps/tutorials/id).

Reviewer: ```suggestion
> * You can find the app ID in the URL when editing the app in [DevTools](https://devtools.bigcommerce.com/). For more information, see [Finding an App's ID](https://developer.bigcommerce.com/api-docs/apps/tutorials/id).
```
